pkg/service/contentclaims/service.go

```diff
@@ -17,22 +19,22 @@ func NewService(indexer types.Service) map[ucan.Ability]server.ServiceMethod[ass
 		assert.Equals.Can(): server.Provide(
 			assert.Equals,
 			func(cap ucan.Capability[assert.EqualsCaveats], inv invocation.Invocation, ctx server.InvocationContext) (assert.Unit, receipt.Effects, error) {
-				log.Errorf("TODO: implement me")
-				return assert.Unit{}, nil, nil
+				err := indexer.PublishClaim(context.TODO(), inv)
+				return assert.Unit{}, nil, err
 			},
 		),
 		assert.Index.Can(): server.Provide(
 			assert.Index,
 			func(cap ucan.Capability[assert.IndexCaveats], inv invocation.Invocation, ctx server.InvocationContext) (assert.Unit, receipt.Effects, error) {
-				log.Errorf("TODO: implement me")
-				return assert.Unit{}, nil, nil
+				err := indexer.PublishClaim(context.TODO(), inv)
+				return assert.Unit{}, nil, err
 			},
 		),
 		assert.Location.Can(): server.Provide(
 			assert.Location,
 			func(cap ucan.Capability[assert.LocationCaveats], inv invocation.Invocation, ctx server.InvocationContext) (assert.Unit, receipt.Effects, error) {
-				log.Errorf("TODO: implement me")
-				return assert.Unit{}, nil, nil
+				err := indexer.PublishClaim(context.TODO(), inv)
```

Comment:
This would be for when we publish our own claims for our own service right?

Cause seperately we have the "CacheClaim" call just for the location commitments manged by others.

Reply:
sorry rewind looks like neither this nor the equals claim are implemented?